homeassistant/components/aftership/sensor.py

```diff
@@ -56,7 +75,40 @@
                       aftership.meta)
         return
 
-    async_add_entities([AfterShipSensor(aftership, name)], True)
+    instance = AfterShipSensor(aftership, name)
+
+    async_add_entities([instance], True)
+
+    async def handle_add_tracking(call):
+        """Call when a user adds a new Aftership tracking from HASS."""
+        title = call.data.get(CONF_TITLE)
+        slug = call.data.get(CONF_SLUG)
+        tracking_number = call.data[CONF_TRACKING_NUMBER]
+
+        await aftership.add_package_tracking(tracking_number, title, slug)
+        await instance.async_update(no_throttle=True)
+
+    hass.services.async_register(
+        DOMAIN,
```

Comment:
We shouldn't register any services under the sensor domain. Register them under the aftership component domain.

Reply:
Updated `DOMAIN` to be `aftership`